### ISSUE TITLE
Transform accounts modal to a page 

### ISSUE THREAD

**Issue by theperu** [RIP-Comm/sossoldi#90]
Currently the lower part of every account modal shows some weird things instead of the right transactions. I think we can re-use a lot of the work that as been done for the transaction page by @GBergatto (#84 )
[Figma link for reference](https://www.figma.com/file/6NyY9yqunpbU7HIkbNEAL3/Sossoldi-App?type=design&node-id=3014-7173&t=ly7oFThllyoSddw1-4)

EDIT: After discussing this issue on discord we decided to remove the modal and use a page instead

**Comment by GBergatto** [comment]
I'm a bit busy in this period but I'll take a look

**Comment by GBergatto** [comment]
I've quickly tried to copy and paste some code from `lib/pages/transactions_page/widgets/list_tab.dart` and it works.

**Comment by GBergatto** [comment]
I've also noticed that we have a few blocks of code that do almost the same thing scattered across the codebase. For example, I've created a widget to display a transaction inside `lib/pages/transactions_page/widgets/transaction_list_tile.dart` and something very similar has been done inside `lib/custom_widgets/transactions_list.dart`.

I believe we should define some rules on how to make widgets more reusable to avoid rewriting the same code over and over and to keep some consistency across different pages. 

Each page should be wrapped inside its own folder and broken down into multiple widgets. If those widgets are specific to that page they should stay inside a `widgets` subfolder. If they are shared across different pages (like the one for transactions) they should stay inside a shared folder like `lib/custom_widgets`
